Allow script-location property to be a list (#67)
patka/cassandra-migration: README.md

  </dependency>
```

> In your properties file you will have new properties that can be set:

Reviewer: ```suggestion
In your properties file you can set the following properties:
```

* cassandra.migration.keyspace-name Specifies the keyspace that should be migrated

> * cassandra.migration.script-locations Overrides the default script location

Reviewer: You should likely restore the original `cassandra.migration.script-location` and mark it as deprecated so people know they need to migrate. Otherwise this change would break backwards compatibility and as such would require a major release (for no good reason).

Author: You're right, thanks for the suggestion
